src/main/java/duke/Greeting.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+
+import duke.ui.UI;
+
+/**
+ * This class encapsulates the functions used to print statements when user enters and exits the program.
+ *
+ */
+public class Greeting {
+
+    /**
+     * This method prints out the line when user start the program
```

Comment:
In method header comments, the first sentence can start in the form `Returns ...`, `Sends ...`, `Adds ...`.
So I guess we need not to say `This method...` because it is already the method's header comment?